packages/Z3/Z3.4.5.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer: [
+  "Gabriel Radanne <drupyog@zoho.com>"
+  "Simon Cruanes <simon.cruanes.2007@m4x.org"
+]
+authors: "Z3 authors"
+homepage: "http://z3prover.github.io/"
+dev-repo: "https://github.com/Z3Prover/z3.git"
+bug-reports: "https://github.com/Z3Prover/z3/issues"
+license: "MIT"
+
+build: [
+  ["python2.7" "scripts/mk_make.py" "--ml" "--prefix" prefix]
+  [make "-C" "build" "-j" jobs]
+]
+install: [
+  [make "-C" "build" "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "Z3"]
+]
+depends: [
+  "ocamlfind" {build}
+  "num"
+  "conf-python-2-7"
+]
+
+messages: [
+  "Before installing any package depending on Z3, please export the following variable:
+  export LD_LIBRARY_PATH=\"$(ocamlfind printconf destdir)/stublibs:${LD_LIBRARY_PATH}\""
```

Comment:
Did you try to get rid of that ? I feel like this is not really acceptable if it gets into opam's main repository.

Reply:
It would be good to get rid of this, indeed. Why is it needed?

Reply:
At the time, it was needed because the package recompile Z3 entirely, Z3's build system was complicated and because I'm not very good at C(++) linking rules.
Now, I think it would be better to rely on system-provided .so/.h files, but it needs modifying Z3's build system or just plain extracting the OCaml part, and building it outside.
opam's LLVM package does the former, so I guess it works.

Reply:
modifying Z3's build system? :-/
this sounds super awful…